ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+Provide a description of your issue.
+For any suggestion, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+For translations, go to [transifex](https://www.transifex.com/supertuxkart/supertuxkart/).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version:
+
+STK Source (ppa, distribution package, official bin,...):
```

Comment:
Seems a bit technical for most users, windows and OSX users especially will not understand what is a "PPA". Maybe make this something like "if you didn't download STK from the official website, please specify where you got it"

Reply:
Ok you added the suggested line, but I would suggest to remove the line about PPA that will confuse a lot of players